converter/src/DelphesEDM4HepConverter.cc

```diff
@@ -579,6 +585,8 @@ namespace k4SimDelphes {
     // relating to curvature (index 2)
     double scale2 = -2.;  // CAREFUL: DELPHES USES THE HALF-CURVATURE
 
+    trackState.omega = cand->C * scale2  ;
```

Comment:
Do you by chance now which delphes version is required for this?

Reply:
I used the latest 'master' of delphes for these tests, but that should be the same with the versions used in the key4hep stack. Also note that this definition of omega is fully in line with the delphes -> edm4hep conversion of the covariance matrix, which is done in the following lines of the code.

Reply:
This should work for us in any case. This has been introduced in delphes since before 3.5.0, which is what we need in any case.